src/listr/tasks/registerMasternodeTaskFactory.js

```diff
@@ -135,14 +124,14 @@ function registerMasternodeTaskFactory(
         options: { persistentOutput: true },
       },
       {
-        title: 'Wait for 15 confirmations',
+        title: 'Wait for 1 confirmation',
```

Comment:
Are you sure that 1 confirmation is enough? We had 15 for some reason. Maybe it's a regtest specific thing?

Reply:
I got it. Thank you.